lib/rules/lint-no-obsolete-elements.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const Rule = require('./base');
+
+const errorMessage = element => `Use of <${element}> detected. Do not use deprecated elements.`;
```

Comment:
I would recommend exporting this error message function and then importing it from the test file to avoid duplication. See `no-negated-condition` rule files for an example.

Reply:
Ya, good idea